libs/model/src/vitest.setup.ts

```diff
@@ -5,6 +5,10 @@ import { bootstrap_testing } from './tester';
 beforeAll(async ({ name }) => {
   const lcsuite = name.includes('-lifecycle');
   if (lcsuite) {
+    if (/[A-Z]/.test(name)) {
```

Comment:
if you are going to use regex, maybe better to only allow lowercase and .

Reply:
But there are a lot more valid characters we can use for database names like numbers or underscores or other things. I am not sure all the possible characters a database could be, so for this reason I think it is best to just exclude the upper case characters, since it cannot be them

Reply:
We own this convention 😆

Reply:
Ah I see. Basically enforce a subset of the pgsql convention. I will do that now.